multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -390,6 +391,18 @@ def get_layer_ids(
 
         return name_to_id
 
+    def get_backbone_layer_names(self):
+        backbone_layer_names = []
+        backbone_layers_patterns = [
```

Comment:
does this cover all backbone layer names used by mmdet?

Reply:
It covers all the common models, i.e. non transformer models use "backbone", transformer based models use "encoder".